core/app_modules/inventory.js

```diff
@@ -103,6 +102,9 @@ inventory.check_out = co.wrap(function*(items, patron, employee) {
 
       var q = 'INSERT INTO checked_out(call, copy, netid, attendant, due)' +
         'VALUES($1, $2, $3, $4, $5)';
+        
+      var notes ='Item checked out by ' + employee + ' due on ' + due.toString().substring(0,10);
```

Comment:
but the date format it gets is from postgres, not javascript, would that change it, I get 
"Item checked out by trevren1 due on 2015-05-21" when I look up the history